CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+All notable changes to this project will be documented in this file.
+
+## [v2.6.1](https://github.com/voxpupuli/puppet-syntax/tree/v2.6.1) (2020-01-11)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-syntax/compare/v2.6.0...v2.6.1)
+
+**Fixed bugs:**
+
+- Add `puppet` gem as runtime dependency [\#116](https://github.com/voxpupuli/puppet-syntax/pull/116) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
I just noticed this was released as a patch release but IMHO this is a breaking change. Before Puppet 4 was allowed and worked. Probably older versions as well. I don't mind dropping support for it, but probably better to do that in at least a minor version but possibly a major. That also means you can drop a lot of code

Reply:
https://github.com/voxpupuli/puppet-syntax/pull/120